compiler/tests/testSuite.bal

```diff
@@ -69,22 +108,29 @@ function compileErrorLineNumber(CompileError err) returns int? {
     }
 }
 
-function listSourcesVPO() returns TestSuiteCases|error => listSources("vpo");
+function listSourcesVPO() returns TestSuiteCases|error => listSources("vpo", "./testSuite");
 
-function listSourcesEU() returns TestSuiteCases|error => listSources("eu");
+function listSourcesEU() returns TestSuiteCases|error => listSources("eu", "./testSuite");
 
-function listSources(string initialChars) returns TestSuiteCases|io:Error|file:Error {
+function listSources(string initialChars, string path) returns TestSuiteCases|io:Error|file:Error {
```

Comment:
This might be already apparent with below comments by James; you only have to list the root bal files from this function, since the import lookup will discover the files inside modules. So you will not need the path to be a parameter.

Reply:
fixed